release-pipeline.yml

```diff
@@ -1710,7 +1713,7 @@ stages:
                     $(Build.ArtifactStagingDirectory)/azCopy-mac-signed/*
                     $(Build.ArtifactStagingDirectory)/azCopy-windows-signed/*
                   changeLogCompareToRelease: 'lastFullRelease'
-                  changeLogType: 'commitBased'
+                  changeLogFile: 'commitBased'
```

Comment:
flag name is different here, is it correct ?

Reply:
updated